web/cookiesCTF/Pages/Correct.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Correct!</title>
+</head>
+	<body>
+		<h1 class="correct">Correct!</h1>
+		<link rel="stylesheet" type="text/css" href="/styles.css">
+		<link rel="preconnect" href="https://fonts.gstatic.com">
+		<link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital@1&display=swap" rel="stylesheet">
```

Comment:
These should be in the `<head>`: https://www.w3schools.com/tags/tag_link.asp (same for the other HTML files)